apollo-router/src/metrics/mod.rs

```diff
@@ -1245,6 +1260,115 @@ macro_rules! assert_histogram_not_exists {
     };
 }
 
+pub(crate) fn count_operation_error_codes(
```

Comment:
As now we're able to properly test our metrics, check for `.with_metrics()` in our codebase can we have tests for this function ?